src/common/snippets/src/lowered/loop_info.cpp

```diff
@@ -250,7 +250,7 @@ void order(const std::vector<size_t>& new_order, std::vector<T>& values) {
                     "Failed to sort values: `new_order` must contain new indexes for ALL values");
     std::vector<T> ordered_values(values.size());
     for (size_t i = 0; i < values.size(); ++i) {
-        ordered_values[new_order[i]] = values[i];
+        ordered_values[i] = values[new_order[i]];
```

Comment:
@a-sidorova Could you please confirm, apply new_order in this manner,  based on [how new_order is generated](https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/src/lowered/loop_manager.cpp#L407-L414). Thank you!

Reply:
Wow, thank very much for this fix! That's right!